### ISSUE TITLE
java.lang.IllegalStateException: containingFile must not be null

### ISSUE THREAD

**Issue by jarst** [aws/aws-toolkit-jetbrains#690]
**Describe the bug**

I have found following stack trace in IDEA event log:

```
java.lang.IllegalStateException: containingFile must not be null
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.isClass(JavaLambdaHandlerResolver.kt:205)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.isInputStreamParameter(JavaLambdaHandlerResolver.kt:200)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.hasRequiredParameters(JavaLambdaHandlerResolver.kt:193)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.isValidHandler(JavaLambdaHandlerResolver.kt:182)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.access$isValidHandler(JavaLambdaHandlerResolver.kt:25)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$findHandlersByClass$2.invoke(JavaLambdaHandlerResolver.kt:151)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$findHandlersByClass$2.invoke(JavaLambdaHandlerResolver.kt:25)
	at kotlin.sequences.FilteringSequence$iterator$1.calcNext(Sequences.kt:135)
	at kotlin.sequences.FilteringSequence$iterator$1.hasNext(Sequences.kt:158)
	at kotlin.sequences.TransformingSequence$iterator$1.hasNext(Sequences.kt:178)
	at kotlin.sequences.SequencesKt___SequencesKt.toCollection(_Sequences.kt:691)
	at kotlin.sequences.SequencesKt___SequencesKt.toSet(_Sequences.kt:732)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.findHandlersByClass(JavaLambdaHandlerResolver.kt:153)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.access$findHandlersByClass(JavaLambdaHandlerResolver.kt:25)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$determineHandlers$1.compute(JavaLambdaHandlerResolver.kt:68)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$determineHandlers$1.compute(JavaLambdaHandlerResolver.kt:25)
	at com.intellij.openapi.project.DumbService.computeWithAlternativeResolveEnabled(DumbService.java:291)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.determineHandlers(JavaLambdaHandlerResolver.kt:66)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$determineHandlers$1.compute(JavaLambdaHandlerResolver.kt:70)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver$determineHandlers$1.compute(JavaLambdaHandlerResolver.kt:25)
	at com.intellij.openapi.project.DumbService.computeWithAlternativeResolveEnabled(DumbService.java:291)
	at software.aws.toolkits.jetbrains.services.lambda.java.JavaLambdaHandlerResolver.determineHandlers(JavaLambdaHandlerResolver.kt:66)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$getIndexer$1$1.visitElement(LambdaHandlerIndex.kt:57)
	at com.intellij.psi.impl.source.tree.java.PsiIdentifierImpl.accept(PsiIdentifierImpl.java:42)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$Companion.acceptLeafNodes(LambdaHandlerIndex.kt:79)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$Companion.acceptLeafNodes(LambdaHandlerIndex.kt:80)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$Companion.acceptLeafNodes(LambdaHandlerIndex.kt:80)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$Companion.access$acceptLeafNodes(LambdaHandlerIndex.kt:69)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$getIndexer$1.map(LambdaHandlerIndex.kt:53)
	at software.aws.toolkits.jetbrains.services.lambda.LambdaHandlerIndex$getIndexer$1.map(LambdaHandlerIndex.kt:20)
	at com.intellij.util.indexing.impl.MapReduceIndex.mapInput(MapReduceIndex.java:276)
	at com.intellij.util.indexing.VfsAwareMapReduceIndex.calculateUpdateData(VfsAwareMapReduceIndex.java:83)
	at com.intellij.util.indexing.impl.MapReduceIndex.update(MapReduceIndex.java:216)
	at com.intellij.util.indexing.FileBasedIndexImpl.updateSingleIndex(FileBasedIndexImpl.java:1668)
	at com.intellij.util.indexing.FileBasedIndexImpl.lambda$doIndexFileContent$27(FileBasedIndexImpl.java:1596)
	at com.intellij.openapi.fileTypes.impl.FileTypeManagerImpl.freezeFileTypeTemporarilyIn(FileTypeManagerImpl.java:522)
	at com.intellij.util.indexing.FileBasedIndexImpl.doIndexFileContent(FileBasedIndexImpl.java:1555)
	at com.intellij.util.indexing.FileBasedIndexImpl.indexFileContent(FileBasedIndexImpl.java:1543)
```

**To reproduce**

Not sure.
All I did was editing existing/adding new java files.
Nothing AWS related withing project and no valid AWS credentials set.

**Your Environment**
 - OS: Widows 8.1
 - JetBrains' Product: IntellijIDEA
 - JetBrains' Product Version: 2018.3.1
 - Toolkit Version: 1.0

**Comment by zhangzhx** [comment]
Thanks for reporting this issue. We'll have a fix in the next release.

**Comment by abrooksv** [comment]
Fixed by #691
